tests/test_api/test_feeds.py

```diff
@@ -137,6 +137,45 @@ def test_post_subscription_validURL_old_feed(self, test_client, user, real_feed)
         check_valid_header_type(r.headers)
         assert r.status_code == 201
 
+    def test_post_subscription_invalid_feed_url_0(self, test_client, user):
```

Comment:
With python-newspaper, we weren’t actually covering all cases of what URLs did/did not work. It’s worth leaving because it better captures all possible input cases until we deal with url parsing and validation and change the behavior.